### ISSUE TITLE
Test randomly failing: Sound actions should trigger 'end' when ended FAILED

### ISSUE THREAD

**Issue by alemangui** [alemangui/pizzicato#10]
The test on 'end' event tigger fail without a clear reproduction path. Travis CI [build 99](https://travis-ci.org/alemangui/pizzicato/builds/138569619) failed and [build 100](https://travis-ci.org/alemangui/pizzicato) passed with exactly the same source code.

@JohnPaulHarold have you reproduced this locally? It might be related to the loading of the sound file used for the test. This test is the first in the suite to load an external file ('base/tests/click.wav') and the timeout parameter may not be large enough in some systems to correctly fetch it. If so, an idea would be to change the timeout parameter of that test (currently 5000ms) to something larger to make sure it has enough time to get the file.

**Comment by alemangui** [comment]
I'm still stumped by this one... the change in timeout didn't have any effect. @JohnPaulHarold can you tell me your node version please?

**Comment by JohnPaulHarold** [comment]
```
C:\Users\John-Paul Harold\Code\pizzicato>node --version
v5.9.0
```

My OSX machine is off being repaired, but I can test on that when it's back.

**Comment by JohnPaulHarold** [comment]
btw, I've tried bumping up the timeout on the tests but I got no joy with that.

**Comment by alemangui** [comment]
Yep, I'm stumped by this one. Travis CI had another episode of failure then success with the exact same code. I'll keep digging.

**Comment by alemangui** [comment]
This is likely solved by commit ca45b5a99ef39455d7e7925561a67eafed622cfb (switching Firefox test environment to Chrome). Will reopen if the issue persists.
